operator/src/main/java/oracle/kubernetes/operator/helpers/EventHelper.java

```diff
@@ -178,6 +200,21 @@ public NextAction onFailure(Packet packet, CallResponse<CoreV1Event> callRespons
               eventData.eventItem.getReason(), eventData.getNamespace());
           return doNext(packet);
         }
+
```

Comment:
This method is getting too long. Please find a way to extract identifiable pieces from it to keep it readable.

Reply:
thanks. done.